lib/usd/translators/meshWriter.h

```diff
@@ -75,12 +74,12 @@ class PxrUsdTranslators_MeshWriter : public UsdMayaPrimWriter
     bool    writeBlendShapeAnimation(const UsdTimeCode& usdTime);
     bool    writeAnimatedMeshExtents(const MObject& deformedMesh, const UsdTimeCode& usdTime);
 
+    /// Used to cache the animated blend shape weight plugs that need to be sampled per-frame.
```

Comment:
We should add the explanation here of why this needs to be a static member.

Reply:
This request has been accommodated for in `2ab03a5`.